src/Bicep.Cli/Services/CompilationService.cs

```diff
@@ -104,8 +107,23 @@ public ParamsSemanticModel CompileParams(string inputPath, bool skipRestore)
             {
                 throw new Exception($"Unable to read file {inputPath}");
             }
+            
+            var paramsFile = SourceFileFactory.CreateBicepParamFile(inputUri, fileText);
+
+            var usingDeclarations = paramsFile.ProgramSyntax.Children.OfType<UsingDeclarationSyntax>();
+            var bicepFilePath = PathHelper.TryGetUsingPath(usingDeclarations.SingleOrDefault(), out var getUsingPathFailureBuilder);
```

Comment:
What are correct ways to utilize ```getUsingPathFailureBuilder``` for error handling? I believe @shenglol helped cathy with this method

Reply:
You can use that delegate to get a more specific exception message and throw that instead of "Path to Bicep file not provided." An example of using the delegate in this way can be found at https://github.com/Azure/bicep/blob/1dcad0c8e0ffe70f243979ad8a37f688a81e9bfa/src/Bicep.Cli/Commands/PublishCommand.cs#L98-L99